ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host1/port-channel-interfaces.yml

```diff
@@ -682,12 +683,15 @@ port_channel_interfaces:
         allowed_vlan: 112
     lacp_fallback_timeout: 5
     lacp_fallback_mode: individual
+    ip_address: dhcp
+    dhcp_client_accept_default_route: true
 
   - name: Port-Channel113
     description: interface_with_mpls_enabled
     switchport:
       enabled: false
     ip_address: 172.31.128.9/31
+    dhcp_client_accept_default_route: true
```

Comment:
Add a comment to state that this won't be rendered because IP address is not DHCP

Reply:
updated